commit 9aebe3eeb6118339298d740a4c567ee5e039e9ee
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Mar 6 16:29:13 2019 +0000

    Revert "staging: vchiq_2835_arm: quit using custom down_interruptible()"
    
    This reverts commit ff5979ad86368425b7da3a25f4e84650b51ff5fd.

diff --git a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_2835_arm.c b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_2835_arm.c
index 9e787b87c0e4..35c91c0f16b6 100644
--- a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_2835_arm.c
+++ b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_2835_arm.c
@@ -570,7 +570,7 @@ create_pagelist(char __user *buf, size_t count, unsigned short type)
 		(g_cache_line_size - 1)))) {
 		char *fragments;
 
-		if (down_killable(&g_free_fragments_sema)) {
+		if (down_interruptible(&g_free_fragments_sema) != 0) {
 			cleanup_pagelistinfo(pagelistinfo);
 			return NULL;
 		}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1189_9aebe3eeb6118339298d740a4c567ee5e039e9ee.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1189_9aebe3eeb6118339298d740a4c567ee5e039e9ee.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
